src/app/backend/resource/pod/detail.go

```diff
@@ -213,6 +213,57 @@ func extractContainerInfo(containerList []v1.Container, pod *v1.Pod, configMaps
 			}
 			vars = append(vars, variable)
 		}
+
+		for _, envFromVar := range container.EnvFrom {
```

Comment:
Export this logic to separate function. Similar what we've done in `evalValueFrom`.

Reply:
It's done, thank you for reviewing.